lib/web/ui/server.go

```diff
@@ -42,6 +42,8 @@ type Server struct {
 	Kind string `json:"kind"`
 	// Tunnel indicates of this server is connected over a reverse tunnel.
 	Tunnel bool `json:"tunnel"`
+	// OpenSSh returns true for OpenSSHNode and OpenSSHEICENodes
```

Comment:
```suggestion
	// OpenSSH returns true for OpenSSHNode and OpenSSHEICENodes
```

But should we just send subkind instead? With this approach, we'll have to add a new bool every time we add a new subkind.

Reply:
Sure, sounds better. I wasn't sure how many subkinds we had but in this case it'll just set us up for the future anyway.